server/src/main/protobuf/protocol.proto

```diff
@@ -49,22 +49,22 @@ message QueryTableRequest {
 
 message ListSharesResponse {
     repeated Share items = 1;
-    optional string next_page_token = 2;
+    optional string nextPageToken = 2;
```

Comment:
This is correct in oss delta sharing server. You can take a look at the result by running

```
curl -XGET https://sharing.delta.io/delta-sharing/shares/delta_sharing/schemas/default/tables?maxResults=1
```